packages/core/src/Tabs/common/types.ts

```diff
@@ -0,0 +1,48 @@
+import { PickSxSize, ComponentPropsWithHTML, Values, PaletteColor } from '@core';
+import { ReactNode, Ref } from 'react';
+import { TABS_SCROLL_POSITIONS, TABS_TYPES } from './constants';
+
+export type TabsType = Values<typeof TABS_TYPES>;
+export type TabsSize = PickSxSize<'small' | 'medium' | 'large'>;
+
+export interface TabsStyleParamsCommon {
+    color: PaletteColor,
+    size: TabsSize,
+}
+
+export interface TabItemWithoutHtml {
+    label: ReactNode,
+    value: string,
+    counter?: ReactNode,
+}
+
+export type TabItem = ComponentPropsWithHTML<TabItemWithoutHtml, 'button'>;
+
+export interface TabsPropsCommon<T extends TabItem = TabItem> {
```

Comment:
можно так, чтоб для `className` и `ref` не писать описание
```suggestion
export interface TabsPropsCommon<T extends TabItem = TabItem> extends Omit<BaseProps, 'permissions'> {
```

Reply:
Добавил